code/__HELPERS/vector.dm

```diff
@@ -0,0 +1,57 @@
+// Basic geometry things.
+
+/vector/
```

Comment:
I assume this is a datum, so could you state the full path here and below if that's true, please?
```suggestion
/datum/vector
```

Reply:
done